static/js/people.js

```diff
@@ -13,6 +13,13 @@ exports.get_by_email = function get_by_email(email) {
     return people_dict.get(email);
 };
 
+exports.get_star_status = function get_star_status(email) {
+    if (page_params.starred_users === undefined) {
+        return false;
+    }
+    return page_params.starred_users.indexOf(email)>=0;
```

Comment:
Okay. Just to ensure that I am getting your idea correctly. You are saying to take `page_params.starred_users` as a key value pair with email as key and it's corresponding value(True or False) as user's starred status. Am I correct?